### DISCUSSION TITLE
Color Win Rate Percentage Point Difference

### DISCUSSION THREAD

**Discussion by FiYir** [unrealities/MTGA_Draft_17Lands#31] [Ideas]
The app currently displays the color win rates for all of the colors in the `Deck Filter` drop-down. The problem with these win rates is that there's no reference, so you can't quickly tell if `WU (54.6%)` means that those colors are good or bad in this set. 

It might be a good idea to add a setting that let's you switch the color win rates in the drop-down to percentage points from the mean, so `WU (54.6%)` would become `WU (-0.4pp)` if the average win rate is 55%.

![image](https://github.com/unrealities/MTGA_Draft_17Lands/assets/151777715/2b4dcf1b-b0db-413d-bd00-330ea23817c3)

What are your thoughts on this? My concern is that it looks a bit too busy.

**Comment by unrealities** [comment]
I like the thought and also agree that I don't think your first shot is the prettiest solution.  Maybe a +/- for each half stdev? 

```
B(52.0%) -
G(62.7%) +++
R(62.3%) ++
U(43.9%) ----
W(66.2%) ++++
BG(53.5%)
BR(54.1%)
GU(55.5%)
GW(56.5%)
RG(59.0%) +
RW(58.9%) +
UB(55.8%)
UR(54.1%)
WB(53.7%)
WU(56.9%)
BRG(55.6%)
BRW(48.7%) --
GWB(51.3%) -
GWU(51.1%) -
RGU(54.8%)
RGW(55.3%)
UBG(51.8%) -
UBR(48.5%) --
WUB(52.1%) -
WUR(52.7%) -
```

**Comment by FiYir** [comment]
@unrealities After thinking about it more, I feel like it makes sense to move the color win rate info from the drop-down to a separate window. This way, we can add the z-score and other metrics without running into space issues.

Also, it might be better to combine the names and colors in the drop-down and get rid of the `Deck Filter Format` setting. For example, instead of just `WR`, it could show as `Boros (WR)`.
